docs/api/api/api.txt

```diff
@@ -1321,6 +1321,17 @@ Parameters:
   see cmd=rebuild
 
 
+POST /build/<project>?cmd=sendsysrq
+
+  Send a single sysrq char to the kernel of a running build
+  Only a subset of debugging requests a are supported (eg. 9, t or w)
```

Comment:
*nitpick* This sounds like there are more. Only "9, t or w" are supported or?

Reply:
some more, but these are the most important ones...

0123456789cltwz and :space: are allowed atm.

Reply:
I just gave examples here, since it might change and I didn't want to have the docu get out of sync in an invalid way .... (it is worse to have invalid docu instead of lacking one;)